config/jobs/cert-manager/config.yaml

```diff
@@ -87,6 +87,12 @@ presets:
   - name: FEATURE_GATES
     value: "AllAlpha=false"
 
+- labels:
+    preset-disable-all-output-formats-enable-feature-gates: "true"
```

Comment:
Nit: this does read like 'disable all output formats and enable feature gates' whilst it does the opposite

Reply:
The naming of these presets is quite painful which suggests something might not be right with this setup I guess..